pull_request_template.md

```diff
@@ -74,15 +74,21 @@
 	- Should be named `Contents`, not `Table of Contents`.
 	- Should be the first section in the list.
 	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
+    - Must not feature `Contributing`, `License` or `Footnotes` sections.
 - Has an appropriate license.
 	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
 		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=master&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
 	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and [Unlicense](https://unlicense.org).
 	- Place a file named `license` or `LICENSE` in the repo root with the license text.
 	- **Do not** add the license name or text to the readme. GitHub already shows the license name at the top of the repo.
+    - A section named `Licence` or `License` pointing to the full license text can still be added at the end of the readme.
```

Comment:
Not sure I actually want that. Feels moot since GitHub already presents the license in the UI.

Reply:
I was just pushing this requirement to be perfectly in sync with the rules enforced by `awesome-lint`.

But apart from that I agree. We can make READMEs leans by getting rid of these `License` sections. Relying on GitHub to cleanly inform users is enough. And the absence of a this section doesn't revoke the licensing in itself as long the license text is present in the repository.

Which policy do you want to enforce @sindresorhus?
1. Add a strict rule to not allow any section named `Licence` or `License` in the README (and eventually raise error in `awesome-lint`).
1. Just remove the item about `Licence` section I added in this PR and keep the status-quo (let user decide upon adding this section or not).

Reply:
1

Reply:
Done: addressed in https://github.com/sindresorhus/awesome/pull/1860/commits/2a1e28da03d55591f914a60b97778bbea57b4f45